### ISSUE TITLE
Calling LogInAsync in new release (10.11.0) Throws Realms.Sync.Exceptions Code 998 (ssl CERT related)

### ISSUE THREAD

**Issue by dgerding** [realm/realm-dotnet#2892]
### What happened?

I've checked twice now. When I update to 10.11.0, the following code suddenly stops working:

 _realmsUser = await App.RealmsSyncApp.LogInAsync(_realmCredentials); 

And instead throws an exception. Here are the details on the exception.

e = {Realms.Sync.Exceptions.AppException: code 998: An unexpected error occurred while sending the request: Authentication failed, see inner exception.: Ssl error:1000007d:SSL routines:OPENSSL_internal:CERTIFICATE_VERIFY_FAILED
  at /Users/builder/jenkins/works...

Reverting to 10.10.0 fixes the problem. 

### Repro steps

Use valid credentials and sign in via RealmsSyncApp.LogInAsync using 10.10.0 Nuget of Realms. 

Update to 10.11.0

Throws error.

### Version

.net standard 2.1

### What SDK flavour are you using?

MongoDB Realm (i.e. Sync, auth, functions)

### What type of application is this?

Xamarin

### Client OS and version

Xamarin Forms Android

### Stacktrace of the exception/crash you're getting

```shell
at Realms.Sync.AppHandle.LogInAsync (Realms.Sync.Native.Credentials credentials) [0x00055] in D:\a\realm-dotnet\realm-dotnet\Realm\Realm\Handles\AppHandle.cs:291 
  at Realms.Sync.App.LogInAsync (Realms.Sync.Credentials credentials) [0x0003d] in D:\a\realm-dotnet\realm-dotnet\Realm\Realm\Sync\App.cs:214 
  at LitFit.XF.App.InitializeExistingUserPersistence (System.String username, System.String pw) [0x000c5] in D:\lf\LitFit.XF\App_Persistence.xaml.cs:179
```

**Comment by nirinchev** [comment]
This looks like a regression with the way we setup the http client. The issue is that the managed http handler is more strict than the native one so it rejects the expired root certificate that we're using to trick older android devices.

**Comment by nirinchev** [comment]
We fixed it and released 10.11.1 that should address it.
